images/operator-registry.yml

```diff
@@ -1,9 +1,10 @@
 content:
   source:
+    dockerfile: operator-registry.Dockerfile
     git:
       branch:
         target: release-{MAJOR}.{MINOR}
-      url: git@github.com:operator-framework/operator-registry.git
+      url: git@github.com:openshift-priv/operator-framework-olm.git
     pkg_managers:
     - gomod
```

Comment:
It looks like we differ in how we specify modules between the registry and OLM image configurations. It's been a while, but it looks like this is using the new format, and the OLM is using the old one? Any reason we can't use the same structure for both?